src/Futhark/Bench.hs

```diff
@@ -40,10 +41,10 @@ import System.Timeout (timeout)
 newtype RunResult = RunResult {runMicroseconds :: Int}
   deriving (Eq, Show)
 
--- | The results for a single named dataset is either an error
--- message, or runtime measurements along the stderr that was
+-- | The results for a single named dataset is either an error message, or
+-- runtime measurements, the number of bytes used, and the stderr that was
 -- produced.
-data DataResult = DataResult String (Either T.Text ([RunResult], T.Text))
+data DataResult = DataResult String (Either T.Text ([RunResult], Map T.Text Int, T.Text))
```

Comment:
Maybe it is time to actually create a data type for this triple.

Reply:
Done. Although, I couldn't really think of a good name... Any suggestions?